pkg/cosign/tuf/client.go

```diff
@@ -336,12 +336,12 @@ func Init(ctx context.Context, altRootBytes []byte, remote client.RemoteStore, t
 			return err
 		}
 	}
-	rootKeys, rootThreshold, err := getRootKeys(altRootBytes)
```

Comment:
hey! so i'm thinking maybe we want to just remove the flag entirely -- right now it was getting the threshold from the current trusted root rather than a threshold out of band like in the flag. This was only because `Init` reuqires a list of root keys and a threshold, but since that's much harder for a user to get (you need to self-parse the root.json) i had just added the whole trusted root as the flag (and then parse for you). WDYT?

otherwise if you would like to keep the flag in case you want more signers than the specified threshold in the root.json, i would say we default to the threshold in the return value here.

Reply:
I was also thinking to just delete the flag but wasn't sure if there is a use case for it. Let's keep it simple and delete it. I will update the patch.